### ISSUE TITLE
Rule `naming-convention` can breaks on anonymous operations

### ISSUE THREAD

**Issue by B2o5T** [dotansimha/graphql-eslint#435]
**Describe the bug**
A simple operation without name will break this rule

**To Reproduce**

```graphql
query { foo }
```
```sh
  ● naming-convention › valid › query { foo }

    TypeError: Cannot read property 'value' of undefined
    Occurred while linting <input>:1

      243 |       } = property;
      244 |       const result = checkNameFormat({
    > 245 |         value: node.value,
          |                     ^
      246 |         style,
      247 |         leadingUnderscore: options.leadingUnderscore,
      248 |         trailingUnderscore: options.trailingUnderscore,

      at checkNode (packages/plugin/src/rules/naming-convention.ts:245:21)
```

**Expected behavior**

Expected to not receive an error as we have already rule `no-anonymous-operations` for that

**Environment:**

- OS: macOS
- `@graphql-eslint/...`: 1.0.0
- NodeJS: 14

**Additional context**
I will make PR to fix this

**Comment by dotansimha** [comment]
Available in v1.0.1. Thanks @B2o5T !
